tests/unit/api/test_metadata_api.py

```diff
@@ -95,22 +113,27 @@ def test_get_current_data_structure_status_as_post(flask_app, mocker):
     spy = mocker.patch.object(
         metadata,
         "find_current_data_structure_status",
-        return_value=MOCKED_DATASTRUCTURE,
+        return_value=MOCKED_DATASTRUCTURES,
     )
     response: Response = flask_app.post(
-        url_for(
-            "metadata_api.get_data_structure_current_status_as_post"),
-        json={"names": "INNTEKT_TJENPEN"},
+        url_for("metadata_api.get_data_structure_current_status_as_post"),
+        json={
+            "names": f"{next(iter(MOCKED_DATASTRUCTURES[0]))},{next(iter(MOCKED_DATASTRUCTURES[1]))}"
```

Comment:
considering the change in the other comment we could do:
```py
"names": ",".join(list(MOCKED_DATASTRUCTURES))
```
or if we want to be more explicit:
```py
"names": ",".join(list(MOCKED_DATASTRUCTURES.keys()))
```